install_config/configuring_authentication.adoc

```diff
@@ -1458,16 +1458,16 @@ within {product-title}:
 So, for example, when:
 
 * <command> = `oc`
-* <version> = `v1.3.0`
+* <version> = The client version. For example, `v3.3.0`. This can change depending on the request is made against the Kubernetes API at `/api`, or the {product-title} API at `/oapi`
```

Comment:
Minor fix: (missing word)
This can change depending on the -> This can change depending on **_if_** the